cli-feed-v3.json

```diff
@@ -482,6 +482,45 @@
         }
       ]
     },
+    "2.11.2": {
+      "Microsoft.NET.Sdk.Functions": "1.0.24",

```

Comment:
FYI -- bumped this up to 24.

Reply:
Cool i will update the templates to use 24 as well. Will post the template version inline